### PR TITLE
fix #80072 medaboutme.ru

### LINKED ISSUE #80072: medaboutme.ru
### Issue URL (Annoyance)
 [https://medaboutme.ru/articles/27_produktov_v_kotorykh_ochen_mnogo_zheleza/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fmedaboutme.ru%2Farticles%2F27_produktov_v_kotorykh_ochen_mnogo_zheleza%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 RC 1
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Strip URLs from tracking parameters
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> EasyList Dutch, <br/>Liste FR, <br/>ROList, <br/>ROLIST2, <br/>road-block, <br/>Frellwit's Swedish Filter, <br/>Dandelion Sprout's Nordic Filters, <br/>AdGuard French, <br/>AdGuard Russian, <br/>AdGuard German
Userscripts: | https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js,<br>https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js,<br>https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by wout-codes**
#80072

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1773,6 +1773,7 @@ revouninstaller.com,logincasino.com,borsagundem.com,officelife.media,kod.ru,mast
 ! ###mc_embed_signup
 kosicednes.sk,tinyprojects.dev,cambio16.com,voxukraine.org,pnas.org,cachevalleydaily.com,bisnis.com,ilpost.it,bsblog.info,alcoparty.com.ua,codastory.com,informazionefiscale.it,kerch.fm,cinemablend.com,prostofinovo.ru,comss.ru,majorgeeks.com,buhgalter.com.ua,providr.com,24tv.ua,babylonbee.com,dexerto.com,football365.com,menrec.com,netdoktor.de,ottawamatters.com,retail-insider.com,walterfootball.com,wweek.com,zozhnik.ru###mc_embed_signup
 !
+medaboutme.ru##.form
```
Comment (stanislav-atr) [inline AnnoyancesFilter/sections/annoyances.txt:1776]: Pretty wide, imo better go for `##.sidebar > form.form`

**Comment by stanislav-atr** [comment]
`##.items-center > a[class*="share-network"]` for share buttons on the top of the article.

File: AnnoyancesFilter/sections/annoyances.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/annoyances.txt:1776]: I agree.

**Comment by wout-codes** [comment]
> ##.items-center > a[class*="share-network-"]

Ok, added.

**Review by stanislav-atr — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**